jetstream/outcomes/firefox_desktop/newtab_do_no_harm.toml

```diff
@@ -0,0 +1,28 @@
+friendly_name = "New Tab - Do No Harm"
+description = """
+Guardrail metrics for the New Tab. Includes user experience and revenue-related metrics for Search, Pocket and AdMarketplace Sponsored Tiles.
+"""
+
+[metrics]
+weekly = [
+    "newtab_searches", 
+    "newtab_ad_clicks",
+    "newtab_homepage_enabled",
+    "newtab_engagement"
+]
+
+overall = [
+    "newtab_searches", 
+    "newtab_ad_clicks",
+    "newtab_homepage_enabled",
+    "newtab_engagement"
+]
```

Comment:
```suggestion
```
Outcomes compute the overall and weekly periods by default for every metric that is referenced. So simply specifying the statistics should be enough